docs/insights/coverage.md

```diff
@@ -0,0 +1,67 @@
+---
+id: coverage
+title: Measuring Your Test Coverage
+sidebar_label: Coverage
+description: See a breakdown of the browser and device variations covered by your tests.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Coverage report allows Org Admins to view and download the breakdown of browsers and mobile devices the tests across their organization have run against. Having insight into your coverage data helps companies:
```

Comment:
The Coverage report, accessible to Org Admins only, lists the browsers and mobile devices on which your organization has tested.

Reply:
I like the way you have phrased this, but I am going to leave it as is because I really want to emphasize the ability to view AND download the report.